app/models/manageiq/providers/amazon/manager_mixin.rb

```diff
@@ -32,6 +32,7 @@ def connect(options = {})
   def verify_credentials(auth_type = nil, options = {})
     raise MiqException::MiqHostError, "No credentials defined" if missing_credentials?(auth_type)
 
+    return true if auth_type == "smartstate_docker"
```

Comment:
@jerryk55 Wasn't this commit merged in PR https://github.com/ManageIQ/manageiq-providers-amazon/pull/341

Reply:
Oh frig yes it was - I thought I'd pulled from master first but obviously not...  Give me a few and I"ll fix it...